networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/topics/AttesterSlashingTopicHandlerTest.java

```diff
@@ -46,7 +51,7 @@
 
   private AttesterSlashingTopicHandler topicHandler =
       new AttesterSlashingTopicHandler(
-          gossipEncoding, dataStructureUtil.randomForkInfo(), validator);
+          gossipEncoding, dataStructureUtil.randomForkInfo(), validator, consumer);
```

Comment:
We should probably verify that the consumer gets called where expected.

Reply:
Done for all 3 new topic handlers.